src/main/groovy/wooga/gradle/github/base/tasks/internal/AbstractGithubTask.groovy

```diff
@@ -164,7 +145,7 @@ abstract class AbstractGithubTask<T extends AbstractGithubTask> extends Conventi
 
     @Internal
     GitHub getClient() {
-        this.clientProvider.get()
+        return clientProvider.get()
```

Comment:
I would prefer if we keep the code style consistent. I know that some developers prefer explicit return statements. In a language which binds every expression to an invisible variable that gets auto returned at function end like groovy does I feel that it becomes just unnecessary characters. Same for scoping with `this`.

To be fair I would like to see what the general public in form of linting rules for groovy has to say about that.

Reply:
I think it is ok to supress return in single line functions like this, I put them only because I'm used to. However, in multi-line functions I think that using `return` may put some extra characters, but it improves readability by a good lot.

Reply:
Changes done, returns removed.